sdk/ai/Azure.AI.Projects/src/Generated/ToolDefinition.cs

```diff
@@ -13,7 +13,7 @@ namespace Azure.AI.Projects
     /// <summary>
     /// An abstract representation of an input tool definition that an agent can use.
     /// Please note <see cref="ToolDefinition"/> is the base class. According to the scenario, a derived class of the base class might need to be assigned here, or this property needs to be casted to one of the possible derived classes.
-    /// The available derived classes include <see cref="AzureAISearchToolDefinition"/>, <see cref="BingGroundingToolDefinition"/>, <see cref="CodeInterpreterToolDefinition"/>, <see cref="MicrosoftFabricToolDefinition"/>, <see cref="FileSearchToolDefinition"/>, <see cref="FunctionToolDefinition"/> and <see cref="SharepointToolDefinition"/>.
+    /// The available derived classes include <see cref="AzureAISearchToolDefinition"/>, <see cref="AzureFunctionToolDefinition"/>, <see cref="BingGroundingToolDefinition"/>, <see cref="CodeInterpreterToolDefinition"/>, <see cref="MicrosoftFabricToolDefinition"/>, <see cref="FileSearchToolDefinition"/>, <see cref="FunctionToolDefinition"/>, <see cref="OpenApiToolDefinition"/> and <see cref="SharepointToolDefinition"/>.
```

Comment:
I see we have added a sample for `AzureFunctionToolDefinition`, can we add a sample for `OpenApiToolDefinition`?

Reply:
Added, however the website we are reaching "wttr.in" is not available due to expired certificate, so I only have checked that it returns a response, but the response itself is not sensible.